lib/rust/flexer/src/automata/dfa.rs

```diff
@@ -16,7 +16,7 @@ use crate::data::matrix::Matrix;
 ///
 /// ```text
 ///  ┌───┐  'D'  ┌───┐  'F'  ┌───┐  'A'  ┌───┐
-///  │ 0 │──────▶│ 1 │──────▶│ 2 │──────▶│ 3 │
+///  │ 0 │ ----> │ 1 │ ----> │ 2 │ ----> │ 3 │
```

Comment:
Why these changes? unicode arrows look so much better!

Reply:
Yes, but they are not monospaced, so they break the alignment of boxes.

Reply:
That is an issue with your font. They are perfectly monospaced on my end.

Reply:
It's an issue of your font as well, because nor mine or github monospaced fonts don't show them aligned at all.